### ISSUE TITLE
Silence being recognized as the word "the" on vosk-model-en-us-0.22

### ISSUE THREAD

**Issue by Roadcrosser** [alphacep/vosk-api#1139]
I'm not sure if it's just me, but I've been having this peculiar issue.

My use-case involves leaving vosk running in the background, so I ran `test_microphone.py` on the `vosk-model-en-us-0.22` model to silence and got results like this:

```
partial: ""
partial: ""
partial: ""
partial: "the"
partial: "the"
partial: "the"
partial: "the"
partial: "the"
...
```

Speaking after that will prepend the `the` as if I started the sentence with it.

The effect is less pronounced on `vosk-model-en-us-0.22-lgraph`:

```
partial: ""
partial: ""
partial: ""
partial: ""
partial: ""
partial: ""
partial: ""
partial: ""
partial: ""
partial: ""
text: "the"
```

This doesn't seem to happen with the `small` variant.

I'd thought there was an issue with my mic, so instead of feeding it my mic input, I'd generated silence with the following method:

```py
np.zeros((8000, 1), dtype=int).tobytes()
```

Only to have the same results.

I'm not sure if this is an issue with the model or just my (hardware?) setup, as I cannot seem to find anyone else having this issue, though I may just be bad at searching.

UPDATE:  Tried on `vosk-model-en-us-0.21`, got this result:

```
partial: ""
partial: ""
partial: ""
partial: ""
partial: ""
partial: ""
partial: ""
partial: ""
partial: ""
partial: ""
text: "hi"
```

**Comment by nshmyrev** [comment]
It is an issue with the particular model version. Same as #837

**Comment by MisterSkrizz** [comment]
I've never used this site before, so excuse my formatting, but I was able to make some headway on this.
My Code looks like this:


import os
import time
import pyaudio
import pyttsx3 as tts
import json
from vosk import Model
from vosk import KaldiRecognizer


model = Model(r"C:\Python\PersonalAssistant\Vosk.0.22")
recognizer = KaldiRecognizer(model, 16000)
mic = pyaudio.PyAudio()
stream = mic.open(format=pyaudio.paInt16, channels=1, rate=16000, input=True, frames_per_buffer=8000)
stream.start_stream()

print("Ready")

while True:
    data = stream.read(8000)
    if recognizer.AcceptWaveform(data):
           print(json.loads(recognizer.Result())['text']) 
    else: 
        pass






All I really did was add the else:pass function. It doesn't include "the" on every line, and just gives a blank line for silent moments. seems to work pretty well, but I'm not sure if this is a band-aid fix or what. I just hope this is helpful :) I got the idea from king129954's post on https://github.com/alphacep/vosk-api/issues/837
